data/sql/updates/pending_db_world/rev_1676897566022575900.sql

```diff
@@ -0,0 +1,9 @@
+--
+CREATE TABLE IF NOT EXISTS `spell_cooldown_overrides` (
```

Comment:
DROP TABLE IF EXISTS `spell_cooldown_overrides`

Reply:
For the record, it doesn't matter because the table is empty so this SQL is safe to be ran twice, the conditional will prevent errors